config/default/uriProvider.conf.php

```diff
@@ -1,11 +1,11 @@
 <?php
+use oat\generis\model\kernel\uri\Bin2HexUriProvider;
+
 /**
  * Default config header
  *
  * To replace this add a file generis/conf/header/uriProvider.conf.php
  */
-
-return new core_kernel_uri_DatabaseSerialUriProvider(array(
-    core_kernel_uri_DatabaseSerialUriProvider::OPTION_PERSISTENCE => 'default',
-    core_kernel_uri_DatabaseSerialUriProvider::OPTION_NAMESPACE => LOCAL_NAMESPACE.'#'
-));
+return new Bin2HexUriProvider([
```

Comment:
This modification will imply to trust this Bin2Hex implementation... Up to u

Reply:
It's used in most production environments already